### ISSUE TITLE
Call to deprecated method setScrollTop

### ISSUE THREAD

**Issue by TobsCore** [defunkt/Zen#105]
Atom displays a warning, that `setScrollTop` is now a view method and should be called via `TextEditorElement::setScrollTop`

```
TextEditor.setScrollTop - /Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:3518:12
<unknown> - /Users/tobscore/.atom/packages/Zen/lib/zen.coffee:152:31
Function.module.exports.Emitter.simpleDispatch - /Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14
Emitter.emit - /Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:129:28
TextEditor.cursorMoved - /Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:2126:27
Selection.markerDidChange - /Applications/Atom.app/Contents/Resources/app.asar/src/selection.js:921:21
```

**Comment by duncdrum** [comment]
Same here: Atom 1.16.0 Zen 0.16.4 macOS 10.12.4

**Comment by elebumm** [comment]
Weird. I got this exact same error using Atom 1.17. I recreated the issue by enable zen mode in the settings view then trying it in a normal text editor window. I am currently using 1.18 and I can't recreate it there. I have forked the repo and I will try and give this a fix.

**Comment by elebumm** [comment]
I created a pull request fixing this issue.

**Comment by braver** [comment]
Fixed in #106 with lots of thanks @ @elebumm

**Comment by elebumm** [comment]
Thanks @braver :smile:
